src/index.ts

```diff
@@ -17,3 +17,5 @@ export const Octokit = Core.plugin([
 ]).defaults({
   userAgent: `octokit-rest.js/${VERSION}`
 });
+
+export type Octokit = InstanceType<typeof Octokit>
```

Comment:
could you add a test that fails without this addition, so we don't break that behavior in future?

Reply:
It looks like https://github.com/octokit/rest.js/blob/master/test/typescript-validate.ts is no longer checked?

Reply:
Oh, I see that was part of the TS rewrite. I'll add it to jest test.